nativelib/src/main/scala/scala/scalanative/runtime/Intrinsics.scala

```diff
@@ -107,7 +111,8 @@ object Intrinsics {
   def storeObject(rawptr: RawPtr, value: Object): Unit = intrinsic
 
   /** Intrinsified computation of derived raw pointer. */
-  def elemRawPtr(rawptr: RawPtr, offset: Long): RawPtr = intrinsic
+  def elemRawPtr(rawptr: RawPtr, offset: Long): RawPtr =
+    intrinsic // TODO(shadaj): word offset
```

Comment:
I'd put TODO in a dedicated issue or a checklist on the PR. TODOs in the code don't age well and we don't use them at all.